### ISSUE TITLE
Update Ver 1.1.0 broken this fix?

### ISSUE THREAD

**Issue by K0MIN0** [darkxex/UnCapFF6PRemaster#2]
Seems to have locked back to 60 despite re-executing the fix, could anyone else confirm if they find the same?

**Comment by tofu-dreg** [comment]
Yes same here, update 1.1.0 have broken all the uncappers (tested FF4 as well).

**Comment by mrgrim** [comment]
I think the correct offset for FF6 1.1.1 is 0x882b1a. I was able to change my framerate editing here.
